packages/components/menu/src/MenuItem/MenuItem.styles.ts

```diff
@@ -1,44 +1,54 @@
 import { css } from 'emotion';
 import tokens from '@contentful/f36-tokens';
+import type { MenuItemProps } from './MenuItem';
 
-export const getMenuItemStyles = () => {
+export const getMenuItemStyles = ({
+  isActive,
+}: {
+  isActive: MenuItemProps['isActive'];
+}) => {
   return {
     root: css({
-      display: 'block',
-      width: '100%',
-      background: 'none',
+      display: 'flex',
+      width: `calc(100% - 2 * ${tokens.spacing2Xs})`,
+      background: isActive ? tokens.gray200 : 'none',
       border: 0,
-      margin: 0,
+      borderRadius: tokens.borderRadiusMedium,
+      margin: `0 ${tokens.spacing2Xs}`,
       outline: 'none',
       fontSize: tokens.fontSizeM,
       lineHeight: tokens.lineHeightM,
-      fontWeight: tokens.fontWeightNormal,
+      fontWeight: isActive ? tokens.fontWeightMedium : tokens.fontWeightNormal,
       position: 'relative',
       textAlign: 'left',
-      padding: `${tokens.spacingXs} ${tokens.spacingM}`,
+      padding: `${tokens.spacing2Xs} ${tokens.spacingXs}`,
       wordBreak: 'break-word',
       whiteSpace: 'break-spaces',
       cursor: 'pointer',
       hyphens: 'auto',
       minWidth: '150px',
       textDecoration: 'none',
-      color: tokens.gray800,
+      color: tokens.gray900,
+
+      '[role="menuitem"] + &': {
+        marginTop: '2px',
+      },
 
       '&:hover': {
         backgroundColor: tokens.gray100,
       },
       '&:focus': {
         boxShadow: `inset ${tokens.glowPrimary}`,
         // just to make boxShadow with rounded corners
-        borderRadius: tokens.borderRadiusMedium,
+        borderRadius: tokens.borderRadiusSmall,
```

Comment:
Any reason for going with small border radius here?
I think it looks odd that it doesn't match the border radius of the element.

<img width="193" alt="Screenshot 2023-03-09 at 14 15 10" src="https://user-images.githubusercontent.com/6163988/224034827-8d0bb918-2609-41eb-a301-6d09c61cdfd3.png">

Reply:
I will defer to @fabe, maybe I misunderstood the comment in Figma 🙂

Reply:
Yeah, we need the small one here to get to the [correct nested border radii](https://www.30secondsofcode.org/articles/s/css-nested-border-radius). So outside is 8px, inside 4px. We could also use our regular border radius (6px), but then the container would need 10px. I think we don't have a token for that?